docs/section23/Rule23-9.md

```diff
@@ -40,16 +41,27 @@
-    - Case 1: the minimum volume flow rate is equal to the maximum of the min_ventilation_flowrate and 50% of the maximum_supply_flowrate - return PASS with a note: `if min_volume_flowrate == max(min_ventilation_flowrate, 0.5 * maximum_supply_flowrate): PASS; note = "The minimum volume flowrate is equal to the maximum of the minimum ventilation flowrate and 50% of the maximum supply flow rate.  If any airflow required to comply with codes or accredidation standards is MORE than this value, the minimum volume airflow should be set to this value.  We are not able to determine the airflow required to comply with codes or accreditation standards at this time, please double check that there are no additional codes or accreditation standards in regards to airflow." `  
-    
-    - Case 2: the minimum volume flow rate is greater than the maximum of the min_ventilation_flowrate and 50% of the maximum_supply_flowrate - return UNDETERMINED with a note: `elif min_volume_flowrate > max(min_ventilation_flowrate, 0.5 * maximum_supply_flowrate): UNDETERMINED; note = "The minimum volume flowrate is greater than the maximum of the minimum ventilation flowrate and 50% of the maximum supply flow rate.  This is correct IF the minimum volume flowrate is equal to any airflow required to comply with codes or accredidation standards, the system passes, otherwise it fails.  We are not able to determine the airflow required to comply with codes or accreditation standards at this time." `  
-    
-    - Case 3: all other cases, the rule fails: `else: FAIL`
+    - Case 1: If the proposed minimum volume flowrate (min_volume_p) is less than or equal tothe maximum of the other two values max(min_ventilation_flowrate and 0.5 * maximum_supply_flowrate), AND the minimum volume flowrate equals the maximum of (min_ventilation_flowrate and 0.5 * maximum_supply_flowrate) then we can provide a definite pass: `if(min_volume_p <= max(min_ventilation_flowrate and 0.5 * maximum_supply_flowrate) and min_volume_flowrate == max(min_ventilation_flowrate, 0.5 * maximum_supply_flowrate, min_volume_p): PASS`
+
+    - Case 3: else if the min_volume_flowrate is less than the maximum of (min_ventilation_flowrate, 0.5 * maximum_supply_flowrate, min_volume_p), then FAIL: `elif min_volume_flowrate < max((min_ventilation_flowrate, 0.5 * maximum_supply_flowrate, min_volume_p)): FAIL`
 
+    - Case 3: Else if proposed minimum volume flow rate is greater than the maximum of (min_ventilation_flowrate and 0.5 * maximum_supply_flowrate), then the minimum flow required by codes & standards might be between the max(min_ventilation_flowrate, 0.5 * maximum_supply_flowrate, min_volume_p) and min_volume_p: UNDETERMINED; note = "The airflow provided in the proposed model is greater than the maximum of the minimum ventilation flowrate and 50% of the maximum supply flowrate.  We cannot determine whether your minimum volume flowrate meets the minimum airflow required to comply with codes or accrediation standards."`
```

Comment:
@KarenWGard I don't think I'm seeing the actual code written out here. Like it's just written in narrative form.